src/PrimaryAuthController.js

```diff
@@ -141,9 +142,14 @@ function (Okta, PrimaryAuthForm, SocialAuth, PrimaryAuthModel, Util, BaseLoginCo
 
       // Add SocialAuth view only when the idps are configured. If configured, 'socialAuthPositionTop'
       // will determine the order in which the social auth and primary auth are shown on the screen.
+      var addButtonSettings = this.model.settings.get('features.addButton');
+
       if (options.settings.get('socialAuthConfigured')) {
         this.add(SocialAuth, {prepend: options.settings.get('socialAuthPositionTop')});
       }
+      if (addButtonSettings && addButtonSettings.visibleWhen) {
```

Comment:
Main concern with that is that we have a divider, that will show up above the button. If we support an array then how do we know if we add the divider or not? Or if we support adding them in different pages? It seems to me we could do that for v2.